templates/community-viz/package.json

```diff
@@ -5,14 +5,15 @@
     "jsFile": "index.js",
     "jsonFile": "index.json",
     "cssFile": "index.css",
-    "manifestFile": "manifest.json"
+    "manifestFile": "manifest.json",
+    "print": "printMessage.js"
```

Comment:
This seems like something that would be better off as a hard-coded constant where it's needed. I don't see the user needing to configure this to be something different.